src/components/PasswordInput/PasswordInput.tsx

```
@@ -15,6 +15,7 @@ type Props = TextFieldProps & {
   value?: string;
   required?: boolean;
   disabledReason?: string;
+  hideStrengthLabel?: boolean;
```

just curious why we'd want to hide "strength:"

---

When I adjusted the grids for the UDF updates, the strength indicator text became squished. We actually already hide that by default globally on mobile so this will give us a way to hide it in instances where the password component may live in a smaller container.